### DISCUSSION TITLE
Show stopwatch name in application

### DISCUSSION THREAD

**Discussion by johncarter-** [spatie/ray#634] [Q&A] [answered]
Is there a way to show the stopwatch name in the Ray application? e.g.

```php
ray()->measure('createOrder');
$this->createOrder();
ray()->measure('createOrder');
```

![image](https://user-images.githubusercontent.com/3776888/151927927-56226ff9-cd7c-4b43-9ed3-4eac5e745292.png)

**Comment by freekmurze** [comment, marked as answer]
You can chain `label` to set a custom label 👍 

<img width="1248" alt="CleanShot 2022-02-01 at 08 31 49@2x" src="https://user-images.githubusercontent.com/483853/151928958-f820b162-2bec-4ba8-9822-953b7da7d537.png">

**Reply by johncarter-** [reply]
Thanks Freek - have a great day mate.
